cmd/binfmt/main.go

```diff
@@ -127,7 +127,28 @@ func parseArch(in string) (out []string) {
 		if err != nil {
 			out = append(out, v)
 		} else {
-			out = append(out, p.Architecture)
+			if p.Architecture == "arm64" {
```

Comment:
This looks wrong when called from `--install` path.

Reply:
we can just add this case to `parseUninstall` where it can it recursively for the second time in case of this specific input.

Reply:
Now using `configs` map to handle this behavior as discussed on Slack.